Source/SwiftLintFramework/Extensions/File+Cache.swift

```diff
@@ -77,7 +77,8 @@ extension File {
         }
         set {
             if newValue {
-                responseCache.values[cacheKey] = Optional<[String: SourceKitRepresentable]>.None
+                let value: [String: SourceKitRepresentable]? = nil
+                responseCache.values[cacheKey] = value
```

Comment:
I've tried just using `responseCache.values[cacheKey] = nil` or `responseCache.values[cacheKey] = .None` but a test started to fail 😬